zyte_common_items/base.py

```diff
@@ -20,6 +20,8 @@
 
 from .util import split_in_unknown_and_known_fields
 
+Breadcrumbs = Optional[str]
```

Comment:
Can we rename it to something else, to avoid confusion with Breadcrumb component?